diff --git a/libfreerdp/codec/dsp_ffmpeg.c b/libfreerdp/codec/dsp_ffmpeg.c
index ef6791400..ea241a29a 100644
--- a/libfreerdp/codec/dsp_ffmpeg.c
+++ b/libfreerdp/codec/dsp_ffmpeg.c
@@ -73,13 +73,15 @@ static BOOL ffmpeg_codec_is_filtered(enum AVCodecID id, BOOL encoder)
 		case AV_CODEC_ID_MP3:
 		case AV_CODEC_ID_ADPCM_MS:
 		case AV_CODEC_ID_G723_1:
+		case AV_CODEC_ID_GSM_MS:
+		case AV_CODEC_ID_PCM_ALAW:
+		case AV_CODEC_ID_PCM_MULAW:
 			return TRUE;
 #endif
 
 		case AV_CODEC_ID_NONE:
 			return TRUE;
 
-		case AV_CODEC_ID_GSM_MS:
 		case AV_CODEC_ID_AAC:
 		case AV_CODEC_ID_AAC_LATM:
 #if !defined(WITH_DSP_EXPERIMENTAL)
